### DISCUSSION TITLE
Time Zone support?

### DISCUSSION THREAD

**Discussion by Chosin137** [JanDeDobbeleer/oh-my-posh#2100] [Troubleshoot] [answered]
I noticed that oh-my-posh uses time.Now to get the time information. Is it possible to display not only the local time but also UTC without changing the Operating System clock?

**Comment by JanDeDobbeleer** [comment, marked as answer]
Yes, as all templates also offer sprig's [functions](http://masterminds.github.io/sprig/date.html), you can use a text segment and the dateInZone function to display a date/time in any time zone.

**Reply by Chosin137** [reply]
Thank you for responding to my question!

`{{ dateInZone .Format .CurrentDate \"UTC\" }}`

The formatting at the end caused me some issues until I got some help from a friend. I couldn't find any examples that showed the `\"UTC\"` I wanted to leave this here just in case anyone else was having trouble adding another time zone.
